SourcetrailExtensionTests/data/TestProjectSettings.props

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <PropertyGroup>
+        <!-- Override system include directories so they're consistent in different environments-->
+        <IncludePath>$(VC_IncludePath);$(WindowsSDK_IncludePath);</IncludePath>
```

Comment:
This is what it's set to on my Win10 machine and what the test data expected anyway. Now the unit tests pass for me, which is a big improvement!